crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -305,10 +305,14 @@ pub struct EncryptionInfo {
 #[derive(Clone, Deserialize, Serialize)]
 pub struct SyncTimelineEvent {
     /// The actual event.
-    pub event: Raw<AnySyncTimelineEvent>,
+    #[serde(rename = "event")]
+    inner_event: Raw<AnySyncTimelineEvent>,
```

Comment:
Naming is a bit weird here: why the `inner_` prefix? Could we name this `raw`?

Reply:
I agree it's a bit of a mess in this transitional state, but since it's all being changed again in [a later commit](https://github.com/matrix-org/matrix-rust-sdk/pull/4082/commits/fde43c93dd402fc0fe243614c62d48cd9168382c#diff-f34dc4bbd38062a031d0c33f7164f950b1735b9b8580d8a6efcb046a9626978cL307-R309), I didn't worry too much about the names here.

Reply:
oh, I'm an idiot, and forgot that Rust allowed you to have fields and methods with the same name, hence the awkward name for this and `inner_encryption_info`. I think this is moot because the field goes away later anyway, but sorry for the unnecessary churn.

Reply:
You're not an idiot.